From e030a19da5a93a8fdca9ed1c7263d6e2be3ceb5a Mon Sep 17 00:00:00 2001
From: Chris Friesen <chris.friesen@windriver.com>
Date: Wed, 5 Aug 2015 18:09:36 -0400
Subject: [PATCH] Increase default error timeouts for iSCSI initiator

The only place we use the iSCSI initiator within our system is on
the compute nodes when using cinder volumes.

Accordingly, change the default values to something that makes more
sense when instance root filesystems are being accessed via iSCSI.

This addresses CGTS-2286.
---
 etc/iscsid.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/etc/iscsid.conf b/etc/iscsid.conf
index b9b3b1f..0f270b5 100644
--- a/etc/iscsid.conf
+++ b/etc/iscsid.conf
@@ -95,7 +95,7 @@ node.leading_login = No
 # - If the value is 0, IO will be failed immediately.
 # - If the value is less than 0, IO will remain queued until the session
 # is logged back in, or until the user runs the logout command.
-node.session.timeo.replacement_timeout = 120
+node.session.timeo.replacement_timeout = 900 
 
 # To specify the time to wait for login to complete, edit the line.
 # The value is in seconds and the default is 15 seconds.
@@ -106,13 +106,13 @@ node.conn[0].timeo.login_timeout = 15
 node.conn[0].timeo.logout_timeout = 15
 
 # Time interval to wait for on connection before sending a ping.
-node.conn[0].timeo.noop_out_interval = 5
+node.conn[0].timeo.noop_out_interval = 900 
 
 # To specify the time to wait for a Nop-out response before failing
 # the connection, edit this line. Failing the connection will
 # cause IO to be failed back to the SCSI layer. If using dm-multipath
 # this will cause the IO to be failed to the multipath layer.
-node.conn[0].timeo.noop_out_timeout = 5
+node.conn[0].timeo.noop_out_timeout = 900 
 
 # To specify the time to wait for abort response before
 # failing the operation and trying a logical unit reset edit the line.
-- 
1.9.1

